lib/underarmour/clients/api_client.rb

```diff
@@ -1,28 +1,43 @@
 module Underarmour
   class ApiClient < BaseClient
-    def initialize(options = {})
+
+    # Kinda weird right now since you have to call it like this... @TODO: see if I can alias some of these class names to shorten these calls
+    # user = Underarmour::ApiClient.scoped_api_client(Underarmour::User, 'user/', access_token: access_token, api_key: api_key)
+    def self.scoped_api_client(class_name, path, options = {})
+      class_name = ApiClient.new(class_name, path, options)
+    end
+
+    # USER_ENDPOINTS = %w(user privacy_option user_profile_photo user_role user_stats)
```

Comment:
These are here just so I don't need to go check the docs for the literal path names